### ISSUE TITLE
neonUtilities in Python: suppress R warnings

### ISSUE THREAD

**Issue by gklarenberg** [NEONScience/NEON-Data-Skills#615]
In the tutorial on using neonUtilities in Python, there is a code block for suppressing R warnings:
````
from rpy2.rinterface import set_writeconsole_warnerror
import logging`
set_writeconsole_warnerror(None)
````
This doesn't work (anymore). It returns the error: `cannot import name 'set_writeconsole_warnerror' from 'rpy2.rinterface' (//anaconda3/lib/python3.7/site-packages/rpy2/rinterface.py)`.

I used the following code to suppress warnings:
```
from rpy2.rinterface_lib.callbacks import logger as rpy2_logger
import logging
rpy2_logger.setLevel(logging.ERROR)   # will display errors, but not warnings
```

System details:
MacOS Mojave 10.14.6
Python 3.7.3
R 4.0.3
JupyterLab 3.3.2 (through Anaconda)

**Comment by cklunch** [comment]
@gklarenberg Thanks Geraldine! Updating this tutorial is already on my list, so this is super helpful.

**Comment by cklunch** [comment]
@gklarenberg Thanks for this tip! Do you run this along with `rpy2` to get it to work correctly, or is this an alternative setup that enables the same behavior as `rpy2`?

**Comment by cklunch** [comment]
The tutorial for using neonUtilities in Python has been updated. https://www.neonscience.org/resources/learning-hub/tutorials/neon-utilities-python

**Comment by gklarenberg** [comment]
@cklunch I installed rpy with `pip install rpy` but I couldn't get `rpy` loaded in jupyter properly. The first line, `import rpy` worked fine, but then `import rpy2.robjects as robjects` threw an error. It kept complaining it could not find my `R_HOME`, and told me to run `python -m rpy2.situation` in the command line - which indeed told me there was no "connection" (or whatever?) to `R_HOME`. R was recognized fine though. The issue is also described here: https://github.com/rpy2/rpy2/issues/796
I tried setting up a new environment with both R and Python (both manually and from the command line) but I just could not get it to work properly. This temporary setting of the environment worked though.
